pages/Portable-Dustbin.md

```diff
@@ -0,0 +1,7 @@
+The Portable Dustbin is an [Item](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Items) that has a unique use.
+
+## Obtaining
+The Portable Dustbin can be crafted in the [Enhanced Crafting Table](https://github.com/TheBusyBiscuit/Slimefun4/wiki/Enhanced-Crafting-Table).
+
+## Usage
+The Portable Dustbin can be used while holding the item by right clicking. Upon right clicking, the item will open up a chest GUI in which any item can be placed, upon the player closing the GUI by pressing ESC, any items placed in the GUI will be deleted and unobtainable. 
```

Comment:
Maybe not mention the key ESC.
Keys can be re-bound and there are many more ways to close an inventory.
Some smartass may think "Oh but the wiki said it'd only do it if I press this specific key.

Reply:
Simply removed "by pressing ESC"
Resolved.